Gemfile

```diff
@@ -48,13 +50,13 @@ group :development, :test do
   gem 'factory_girl_rails'
   gem 'pry-rails'
   gem 'rspec-rails', '~> 3.3.0'
+  gem 'apivore', git: 'https://github.com/westfieldlabs/apivore.git'
```

Comment:
yeah it does. I guess that's good enough. thought it might be nice to know when we can move back to a version rather than replying on source code, but we can do that via gemfile lock